README.md

```diff
@@ -89,6 +89,9 @@ The commands to build the Locale are started from the main directory of the proj
 make clean
 make -e SPHINXOPTS="-D language='de'" BUILDDIR="build/de" html
 ```
+"""
+in windows: sphinx-build source build/de -D language='de' -b html
```

Comment:
Better to write "in windows" in plain text above the code snippet. 
The code snippet should only contain the command itself. That way it will later be clearer what needs to be copied.

Reply:
changed - now identical to the command directly above